SCRIPTS/resource.VCFparser.pl

```diff
@@ -883,14 +974,12 @@
 	
 	### Closing input file
 	close IN;
-	
 } elsif ( $reference eq "1Gp3" and ( $population eq "PAN" or $population eq "EUR" or $population eq "AFR" or $population eq "AMR" or $population eq "EAS" or $population eq "SAS" ) ) {
```

Comment:
Please remove:

```
elsif ( $reference eq "1Gp3" and ( $population eq "PAN" or $population eq "EUR" or $population eq "AFR" or $population eq "AMR" or $population eq "EAS" or $population eq "SAS" ) ) {
 	die " *** ERROR *** Parsing of 1000G phase 3 type references is not available yet. We need to
 think about how to handle multi-allelic variants. One option is to split the fields in to
 multiple rows, one for each variant. The variantID should as a consequence be chr<#>:<#>:MinorAllele_MajorAllele.\n";
```

Since you now added this option.

Reply:
@xEmz did you check this ?